src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/WithdrawalApplications.cshtml

```diff
@@ -5,11 +5,11 @@
 @model SFA.DAS.AdminService.Web.ViewModels.Apply.Applications.ApplicationsDashboardViewModel
 
 @{
-    ViewBag.Title = "Organisation withdrawal applications";
+    ViewBag.Title = "Withdrawal applications";
     Layout = "_Layout";
 }
 
-<a asp-action="Index" asp-controller="Dashboard" class="govuk-back-link">Return to dashboard</a>
+<a asp-action="Index" asp-controller="Dashboard" class="govuk-back-link">Back</a>
```

Comment:
There is an AC for this which does not mention it changing on each story

Reply:
It was changed to back in the content deck but I've confirmed it should be 'Return to dashboard' so I've reverted